frontend/components/common/Layout/Footer.tsx

```diff
@@ -164,17 +162,23 @@ export function Footer({ stores, currentStore, globalSettings }: FooterProps) {
                                  position="relative"
                                  p="0"
                               >
-                                 <Image
-                                    layout="fill"
-                                    objectFit="contain"
-                                    src={
-                                       partner.image?.url || placeholderImageUrl
-                                    }
-                                    alt={
-                                       partner.image?.alternativeText ||
-                                       undefined
-                                    }
-                                 />
+                                 {partner.image?.url ? (
```

Comment:
Out of curiosity, why did you choose to change this section to a ternary instead of what you had previously?

Reply:
Ahhh gotcha!